src/OrchardCore.Modules/OrchardCore.Media.Azure/ImageSharpImageCacheOptions.cs

```diff
@@ -0,0 +1,38 @@
+namespace OrchardCore.Media.Azure;
+
+public class ImageSharpImageCacheOptions
+{
+    /// <summary>
+    /// The Azure Blob connection string.
+    /// </summary>
+    public string ConnectionString { get; set; }
+
+    /// <summary>
+    /// The Azure Blob container name.
+    /// </summary>
+    public string ContainerName { get; set; }
+
+    /// <summary>
+    /// Create blob container on startup if it does not exist.
+    /// </summary>
+    public bool CreateContainer { get; set; } = true;
+
+    /// <summary>
+    /// The base directory path to use inside the container for this store's content.
+    /// </summary>
+    public string BasePath { get; set; } = "";
+
+    /// <summary>
+    /// Remove blob container on tenant removal if it exists.
+    /// </summary>
+    public bool RemoveContainer { get; set; }
+
+    /// <summary>
+    /// Returns a value indicating whether the basic state of the configuration is valid.
+    /// </summary>
+    /// <returns></returns>
```

Comment:
Once a smart man requested me to remove empty docs tags during review :)

Reply:
And he was absolutely right :D. I don't know how I left these in.